Source/ScreenRotate.spoon/init.lua

```diff
@@ -0,0 +1,113 @@
+--- === ScreenRotate ===
```

Comment:
could the name for this change to `ToggleScreenRotation`? It would be more specific about what it does.